target/linux/ramips/image/mt7621.mk

```diff
@@ -121,6 +121,16 @@ define Device/mediatek_ap-mt7621a-v60
 endef
 TARGET_DEVICES += mediatek_ap-mt7621a-v60
 
+define Device/xzwifi_creativebox-v1
+  DTS := CreativeBox-v1
+  IMAGE_SIZE := $(ralink_default_fw_size_32M)
+  DEVICE_TITLE := CreativeBox v1
+  DEVICE_PACKAGES := \
+	kmod-ata-core kmod-ata-ahci kmod-mt7603 kmod-mt76x2 kmod-sdhci-mt7620 \
+	kmod-usb3
```

Comment:
usb3? Not mentioned in the commit message.

Reply:
Hardware spec cont.

Interfaces:
GBE RJ45 x5
USB3.0 x1
eSATA(with USB2.0) x1
SATA x1
UART x1
I2C x1
JTAG x1
SPI x2 (one empty)